### PR TITLE
Fix #7289: Add 1.50 translations

### LINKED ISSUE #7289: Add 1.50 translations

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #7289 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Test Plan:

Verify that custom filter lists screen is translated https://github.com/brave/brave-ios/pull/6857

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: App/BraveWidgets/tr.lproj/BraveWidgets.strings [hunk 1]
```diff
@@ -64,6 +64,9 @@
 /* (No Comment) */
 "e13lS3-vs094F" = "Sadece onaylamak için soruyorum, \"Yeni Gizli Sekme\" bölümünü mü istediniz?";
 
+/* (No Comment) */
+"e13lS3-XCyA8c" = "Sadece onaylamak için soruyorum, \"Brave News\" bölümünü mü istediniz?";
```
Comment (soner-yuksel) [inline App/BraveWidgets/tr.lproj/BraveWidgets.strings:68]: Is this normal this 3 sentence repeats in this section ?

**Review by soner-yuksel — APPROVED** [review]

File: App/BraveWidgets/zh-TW.lproj/BraveWidgets.strings [hunk 2]
```diff
@@ -232,6 +247,9 @@
 /* (No Comment) */
 "xcI3h9" = "已封鎖廣告";
 
+/* (No Comment) */
+"XCyA8c" = "Brave News";
```
Comment (nuo-xu) [inline App/BraveWidgets/zh-TW.lproj/BraveWidgets.strings:251]: this is not translated? 
"Brave News" -> "Brave 新聞"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 3]
```diff
@@ -19,6 +19,15 @@
 /* Label to display in the Discoverability overlay for keyboard shortcuts */
 "addBookmarkTitle" = "添加书签";
 
+/* A title within a cell where a user can navigate to an add screen. */
+"AddCustomFilterList" = "新增自定义筛选器列表";
+
+/* A description of a section in a list that allows you to add custom filter lists found in the footer of the add custom url screen */
+"AddCustomFilterListDescription" = "新增由您信任的社群所建立及維護的額外清單。";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "新增由您信任的社群所建立及維護的額外清單。" -> "新增由您信任的社群所建立维护的额外清单。"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 4]
```diff
@@ -814,12 +829,39 @@
...
+"FilterListsDownloadFailed" = "下載失敗";
+
+/* If a filter list is not yet downloaded this label shows up instead of a last download date, signifying that the download is still pending. */
+"FilterListsDownloadPending" = "等待下载";
+
+/* This is a button on the top navigation that takes the user to an add custom filter list url to the list */
+"FilterListsEdit" = "编辑";
+
+/* This is a placeholder for an input field that takes a custom filter list URL. */
+"FilterListsEnterFilterListURL" = "輸入過濾器清單 URL";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "輸入過濾器清單 URL" -> "输入过滤器清单 URL"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 5]
```diff
@@ -28,6 +31,15 @@
...
 "wallet.activateToken" = "激活令牌 %@";
 
+/* The description when showing no transactions inside Activity tab. */
+"wallet.activityPageEmptyDescription" = "在您的首次交易后，您将能够在这里查看它。";
+
+/* The title when showing no transactions inside Activity tab. */
+"wallet.activityPageEmptyTitle" = "暂无交易";
+
+/* The title of the tab that will display user's transaction activity for all accounts. */
+"wallet.activityPageTitle" = "活動";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "活動" -> "活动"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 6]
```diff
@@ -550,6 +568,24 @@
 /* The button title for showing the screen to change what assets are visible */
 "wallet.editVisibleAssetsButtonTitle" = "编辑可见资产";
 
+/* Title on the button that users can click to enable Brave to resolve the ENS domain they entered. */
+"wallet.ensDomainInterstitialPageButtonProceed" = "使用 Infura 伺服器繼續操作";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "使用 Infura 伺服器繼續操作" -> "使用 Infura 服务器继续操作"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 7]
```diff
@@ -550,6 +568,24 @@
...
+
+/* Description displayed when users chose Brave to ask them if they want the ENS to be resolved every time they enter one. The first '%@' will be replaced with a link to Infura's terms of use page. The second '%@' will be replaced with the value of 'Web3DomainInterstitialPageTAndU'. The third '%@' will be replaced with a link to Infura's privacy policy page. The last '%@' will be replaced with the value of 'Web3DomainInterstitialPagePrivacyPolicy'. */
+"wallet.ensDomainInterstitialPageDescription" = "Brave 将使用 Infura 来解决以太坊名称服务（ENS）的 .eth 域名。Brave 将隐藏你的 IP 地址。如果你启用这个功能，Infura 将看到有人试图访问这些 .eth 域名，但没有其他内容。查看 Infura 的 <a href=%1$@>%2$@</a> 和 <a href=%3$@>%4$@</a>。";
+
+/* Title displayed when users chose Brave to ask them if they want the ENS domain to be resolved every time they enter one. */
+"wallet.ensDomainInterstitialPageTitle" = "在 Brave 中啟用支援以太坊域名服務 (ENS)？";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "在 Brave 中啟用支援以太坊域名服務 (ENS)？" -> "在 Brave 中启用支援以太坊域名服务 (ENS)？"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 8]
```diff
@@ -550,6 +568,24 @@
...
+
+/* Title on the button that users can click to enable Brave to resolve the ENS offchain lookup for the domain they entered. */
+"wallet.ensOffchainDomainInterstitialPageButtonProceed" = "繼續鏈下查找";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "繼續鏈下查找" -> "继续链外查找"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 9]
```diff
@@ -550,6 +568,24 @@
...
+
+/* Description displayed when users chose Brave to ask them if they want the ENS to be resolved every time they enter one. The first '%@' will be replaced with a link to Brave Wallet's offchain lookup privacy considerations. The second '%@' will be replaced with the value of 'ensOffchainDomainInterstitialLearnMore'. */
+"wallet.ensOffchainDomainInterstitialPageDescription" = "这个 .eth 域名被存储在链外，将由第三方网关解析。如果你启用 ENS 链外查询，第三方网关将看到你正试图访问 .eth 域名。<a href=%1$@>%2$@</a>。";
+
+/* Title displayed when users chose Brave to ask them if they want the ENS Offchain domain to be resolved every time they enter one. */
+"wallet.ensOffchainDomainInterstitialPageTitle" = "在 Brave 中啟用以太坊域名服務 (ENS) 鏈下查找支援？";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "在 Brave 中啟用以太坊域名服務 (ENS) 鏈下查找支援？" -> "在 Brave 中启用以太坊域名服务 (ENS) 链外查找支援？"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 10]
```diff
@@ -1552,13 +1618,22 @@
...
+"wallet.web3DomainInterstitialPageButtonDisable" = "禁用";
+
+/* Hyper link copy displayed as part of 'snsDomainInterstitialPageDescription'. It will redirect user to the privay policy webpage of the server that Brave uses to resolve SNS domain. */
+"wallet.web3DomainInterstitialPagePrivacyPolicy" = "隐私政策";
+
+/* Hyper link copy displayed as part of 'snsDomainInterstitialPageDescription'. It will redirect user to the 'terms of use' webpage of the server that Brave uses to resolve SNS domain. */
+"wallet.web3DomainInterstitialPageTAndU" = "使用条款";
+
+/* One of the options for Brave to handle Ethereum/Solana Name Service domain name and IPFS scheme url. 'Ask' means Brave will ask user first before enable or disable resolving ENS/SNS domain name and IPFS scheme url. */
 "wallet.web3DomainOptionAsk" = "詢問";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: not from this change but old one is wrong "詢問" -> "询问"

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 11]
```diff
@@ -1567,6 +1642,18 @@
...
 "wallet.web3DomainOptionsTitle" = "解析 Solana 名称服务 (SNS) 域名";
 
+/* A brief explaination of IPFS on privacy. The first '%@' will be replaced with a link to a Brave article that explains how does IPFS impact users' privacy in details. The second '%@' will be replaced with 'web3IPFSInterstitialIPFSPrivacyLearnMore'. */
+"wallet.web3IPFSInterstitialIPFSPrivacy" = "IPFS（InterPlanetary File System）是用于点对点数据存储和传递的公共网络。这个协议和 HTTP(S) 具有不同的隐私考虑。<a href=%1$@>%2$@</a>";
+
+/* A brief explaination of resolving IPFS scheme url using a public gateway. */
+"wallet.web3IPFSInterstitialIPFSPublicGateway" = "保护你的 IP 地址和你所请求的内容不被潜在的第三方观察者发现，但不受公共网关运营商的影响。 网关运营商负责确保通过网关提供给你的内容是你请求的内容。另外，你的设备不参与公共 IPFS 网络。";
+
+/* The title of the IPFS interstitial page for user to set the preference of IPFS scheme url handling. */
+"wallet.web3IPFSInterstitialIPFSTitle" = "設定您的 IPFS 喜好設定";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "設定您的 IPFS 喜好設定" -> "设定您的 IPFS 喜好设定"

**Review by nuo-xu — CHANGES_REQUESTED** [review]
not sure why most of the chinese(simplified) got chinese(traditional) translation.

File: App/BraveWidgets/zh-TW.lproj/BraveWidgets.strings (hunk 2 above)
Comment (iccub) [reply App/BraveWidgets/zh-TW.lproj/BraveWidgets.strings:251]: we do not translate it afaik

**Comment by iccub** [comment]
@nuo-xu all your fixes here https://github.com/brave/brave-ios/pull/7291/commits/7546d2043694b4da3547e97744b65249917820bc

File: App/BraveWidgets/tr.lproj/BraveWidgets.strings (hunk 1 above)
Comment (iccub) [reply App/BraveWidgets/tr.lproj/BraveWidgets.strings:68]: something xcode generated, like with intents?

Comment (soner-yuksel) [reply App/BraveWidgets/tr.lproj/BraveWidgets.strings:68]: No idea tbh, there is nothing with translation though.

File: Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings [hunk 12]
```diff
@@ -1489,6 +1543,18 @@
 /* The short name of one of the on ramp provider. */
 "wallet.transakProviderShortName" = "Transak";
 
+/* Description displayed when users chose Brave to ask them if they want the Unstoppable Domains to be resolved every time they enter one. The first '%@' will be replaced with a list of supported TLDs like '.x' or '.bitcoin'. The second '%@' be replaced with a link to Infura's terms of use page. The third '%@' will be replaced with the value of 'Web3DomainInterstitialPageTAndU'. The fourth '%@' will be replaced with a link to Infura's privacy policy page. The last '%@' will be replaced with the value of 'Web3DomainInterstitialPagePrivacyPolicy'. */
+"wallet.udDomainInterstitialPageDescription" = "Brave 将使用 Infura 来解析 Unstoppable Domains 上的.crypto（还有 %1$@）域名。Brave 将隐藏你的 IP 地址。如果你启用这个功能，Infura 将看到有人试图访问这些域名，但没有其他内容。查看 Infura 的 <a href=%2$@>%3$@</a> 和 <a href=%4$@>%5$@</a>。";
+
+/* Title displayed when users chose Brave to ask them if they want the Unstoppable Domains domain to be resolved every time they enter one. */
+"wallet.udDomainInterstitialPageTitle" = "在 Brave 中啟用支援 Unstoppable Domains？";
```
Comment (nuo-xu) [inline Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings]: "在 Brave 中啟用支援 Unstoppable Domains？" -> "在 Brave 中启用支援 Unstoppable Domains？"

**Review by nuo-xu — APPROVED** [review]

**Pull request merged**